ubyssey/templates/article/blank.html

```diff
@@ -7,7 +7,7 @@
 {% block content %}
     <main class="page">
         <div class="u-container u-container--content">
-          {{ page.get_html|safe }}
+          {{ page.html|safe }}
```

Comment:
I could be wrong, but you shouldn't have to add the `|safe` filter to this anymore